tools/make/kube.mk

```diff
@@ -1,5 +1,7 @@
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION ?= 1.24.1
+# CEL_VALIDATION_K8S_VERSION refers to the version of kubebuilder assets to be downloaded in validation test.
```

Comment:
🙈 
please raise a GH issue to track this work around

Reply:
it's fixed, hope CI is happy.